### PR TITLE
0.10.5

### PR DESCRIPTION
**Pull request by flarnie**
It's time for a release~! 👐✨🦋

### CODE REVIEW

**Comment by flarnie** [comment]
I'm shipping this asap - feel free to request changes to the `CHANGELOG` in follow-up issues/PRs.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,42 @@ Notable changes to Draft.js will be documented in this file.
...
+* Cause editor to break out of code block when user enters two blank lines. (Hanzhi Zhang
+  in [548fd5d1](https://github.com/facebook/draft-js/commit/548fd5d1b1c31b7b4c79cd70b101fae69d522b3f))
+
+### Fixed
+* Preserve list indentation when copying and pasting from one Draft.js editor
+  into another. ([@GordyD](https://github.com/GordyD) in [#1605](https://github.com/facebook/draft-js/pull/1605))
+* Fix `cannot read property 'update' of undefined` error that was thrown when
+  typing same character into selection that starts with that character. ([@existentialism](https://github.com/existentialism) in
+  [#1512](https://github.com/facebook/draft-js/pull/1512))
+* Fix `encodeRawBlocks` to handle non-contiguous entities. Entities should
```
Comment (existentialism) [inline CHANGELOG.md]: nit: Entities should always be `contigious -> contiguous`...

Comment (flarnie) [reply CHANGELOG.md]: Thanks! Will fix in a follow-up - don't want to make CI restart, it is already taking forever-and-a-half.

Comment (existentialism) [reply CHANGELOG.md]: Sounds good!

**Pull request closed without merging**